src/Workspaces/Core/Portable/Workspace/Solution/TextLoader.cs

```diff
@@ -184,10 +226,13 @@
                 _filePath = filePath;
             }
 
-            public override Task<TextAndVersion> LoadTextAndVersionAsync(Workspace workspace, DocumentId documentId, CancellationToken cancellationToken)
-                => Task.FromResult(LoadTextAndVersionSynchronously(workspace, documentId, cancellationToken));
+            internal override string? FilePath
+                => _filePath;
+
+            public override Task<TextAndVersion> LoadTextAndVersionAsync(LoadTextOptions options, CancellationToken cancellationToken)
+                => Task.FromResult(LoadTextAndVersionSynchronously(options, cancellationToken));
 
-            internal override TextAndVersion LoadTextAndVersionSynchronously(Workspace workspace, DocumentId documentId, CancellationToken cancellationToken)
+            internal override TextAndVersion LoadTextAndVersionSynchronously(LoadTextOptions options, CancellationToken cancellationToken)
```

Comment:
>options [](http://example.com/codeflow?start=93&length=7)

The parameter is getting ignored. Is this the right thing to do?

Reply:
Yes, it is. This loader does not support reloading the text as it gets it from the `_container` and not from file/stream. Therefore it can't apply the checksum algorithm specified in the options.